src/server/services/wms/qgswms.cpp

```diff
@@ -34,7 +34,7 @@
 #include "qgswmsparameters.h"
 
 #define QSTR_COMPARE( str, lit )\
-  (str.compare( QLatin1String( lit ), Qt::CaseInsensitive ) == 0)
+  (str.compare( QStringLiteral( lit ), Qt::CaseInsensitive ) == 0)
```

Comment:
I see you are changing these all recently. I remember having read that `QLatin1String` is preferable where overloads for it exist. Has that changed or was that never true?

Reply:
I'm not sure, I've read this: https://www.qt.io/blog/2014/06/13/qt-weekly-13-qstringliteral apparently it's more efficient whenever it must be converted to QString, but you are right, it's probably be more efficient in case of overloads.

Reply:
reverted